tests/test_utils_signal.py

```diff
@@ -2,20 +2,20 @@
 from twisted.trial import unittest
 from twisted.python.failure import Failure
 from twisted.internet import defer, reactor
-from pydispatch import dispatcher
 
+from scrapy.dispatch import Signal
 from scrapy.utils.signal import send_catch_log, send_catch_log_deferred
 
 
 class SendCatchLogTest(unittest.TestCase):
```

Comment:
This test should probably be moved to a new module `test_dispatcher` or something, since the real code no longer lives in `utils.signal`